libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -35,6 +39,50 @@ const getIcon = (content) => {
   return icons.company;
 };
 
+const showTooltip = (element, message, time = 5000) => {
+  element.setAttribute('data-pep-dismissal-tooltip', message);
+  const cleanup = () => {
+    element.removeAttribute('data-pep-dismissal-tooltip');
+  };
+  const id = setTimeout(() => {
+    cleanup();
+  }, time);
+  element.addEventListener('click', () => {
+    cleanup();
+    clearTimeout(id);
+  }, { once: true });
+};
+
+const playFocusAnimation = (element, iterationCount = 2, animationDuration = 2500) => {
```

Comment:
Defaults should reflect the `CONFIG` values

Reply:
The `CONFIG` values are defaulted into `this.options`, but I think it doesn't hurt to add them here too.